app/models/super_good/solidus_taxjar/transaction.rb

```diff
@@ -0,0 +1,7 @@
+module SuperGood
+  module SolidusTaxjar
+    class OrderTransaction < ActiveRecord::Base
+      belongs_to :order, class_name: "Spree::Order"
+    end
+  end
+end
```

Comment:
We need to move this file to `order_transaction.rb` in order to appease Zeitwerk's autoloading behaviour.

Reply:
Fixed! Good Catch